modules/complex-number/test/test_grechuhin_nikta_complex_number.cpp

```diff
@@ -0,0 +1,98 @@
+// Copyright 2017 Grechuhin Nikita
+
+#include <gtest/gtest.h>
+
+#include "include/complex_number.h"
+
+TEST(Grechuhin_Nikita_Complex_Number_Test, Can_Create_Complex_Number) {
+    // Arrange
+    double re = 10.1;
+    double im = 3.196;
+
+    // Act
+    ComplexNumber z(re, im);
+
+    // Assert
+    EXPECT_EQ(re, z.getRe());
+    EXPECT_EQ(im, z.getIm());
+}
+
+TEST(Grechuhin_Nikita_Complex_Number_Test, Can_Create_By_Copy) {
+    // Arrange
+    double re = 5.4;
+    double im = 1.3;
+
+    // Act
+    ComplexNumber z(re, im);
+    ComplexNumber test_number(z);
+
+    // Assert
+    EXPECT_EQ(test_number, z);
+}
+
+TEST(Grechuhin_Nikita_Complex_Number_Test, Can_Set_Re_and_Im)
+{
```

Comment:
remove brace to the prev line
SEE THE LOG OF TRAVIS!
.cpp:34:  { should almost always be at the end of the previous line  [whitespace/braces] [4]